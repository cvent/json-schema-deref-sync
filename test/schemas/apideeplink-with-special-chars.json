[
  {
    "description": "Remove a widget",
    "href": "/widgets/{widgetId}",
    "method": "DELETE",
    "rel": "delete",
    "title": "Delete"
  },
  {
    "description": "Update a widget.",
    "href": "/widgets/{widgetId}",
    "method": "POST",
    "rel": "update",
    "title": "update",
    "schema": {
      "properties": {
        "foo-id": {
          "$ref": "widget-update.json#/properties/foo~1bar~1id"
        },
        "email": {
          "$ref": "widget-update.json#/properties/email~0address"
        }
      },
      "required": [
        "fooId",
        "email"
      ],
      "type": "object"
    }
  },
  {
    "description": "Invite account user to an account.",
    "href": "/invite/account/:accountId",
    "method": "POST",
    "rel": "parent-action",
    "title": "invite-account-user"
  }
]