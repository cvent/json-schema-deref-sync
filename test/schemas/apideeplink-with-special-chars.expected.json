[
  {
    "description": "Remove a widget",
    "href": "/widgets/{widgetId}",
    "method": "DELETE",
    "rel": "delete",
    "title": "Delete"
  },
  {
    "description": "Update a widget.",
    "href": "/widgets/{widgetId}",
    "method": "POST",
    "rel": "update",
    "title": "update",
    "schema": {
      "properties": {
        "foo-id": {
          "description": "Unique identifier.",
          "readOnly": true,
          "format": "uuid",
          "example": "01234567-89ab-cdef-0123-456789abcdef",
          "type": "string",
          "minLength": 1
        },
        "email": {
          "description": "Email",
          "format": "email",
          "readOnly": false,
          "type": "string",
          "minLength": 1
        }
      },
      "required": [
        "fooId",
        "email"
      ],
      "type": "object"
    }
  },
  {
    "description": "Invite account user to an account.",
    "href": "/invite/account/:accountId",
    "method": "POST",
    "rel": "parent-action",
    "title": "invite-account-user"
  }
]